src/demo/demo.js

```diff
@@ -172,38 +171,48 @@ const DummyHostApp = () => (
   </div>
 )
 
-let container
-window.addEventListener('message', (event) => {
-  if (event.data === 'init' && !port2) {
-    port2 = event.ports[0]
-    port2.onmessage = onMessage
+const renderDemoApp = () => {
+  /* if (process.env.NODE_ENV === 'test') {
+    return
+  } */
```

Comment:
Is this commented out for now because the Demo App mount test is being skipped & that test is meant to mock `window.Onfido`?

Reply:
This check leads to UI tests breaking. I'm figuring out a solution for it

Reply:
You've probably tried it already, but could it be that we need to return an element here even if it's just `<div></div>`?

Reply:
Nope, actually the original attempt here is to disable auto rendering when import `demo.js` from the unit tests, but because unit tests & UI tests use the same `NODE_ENV=test` so in UI tests we don't have anything rendered. One possible way is that we could export all the components from a `demo/components.js` file and import it from `demo.js` and unit test files.

Reply:
Updated on this: we can safely remove this part: https://github.com/onfido/onfido-sdk-ui/pull/1255#discussion_r533557915